packages/optimizers/svgo/src/SVGOOptimizer.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import {Optimizer} from '@parcel/plugin';
+import ThrowableDiagnostic from '@parcel/diagnostic';
+import {blobToString} from '@parcel/utils';
+
+import * as svgo from 'svgo';
+import path from 'path';
+
+export default (new Optimizer({
+  async loadConfig({config}) {
+    let configFile = await config.getConfig([
+      'svgo.config.js',
+      'svgo.config.json',
+    ]);
+
+    if (configFile) {
```

Comment:
What happens when `if (configFile)` evaluates to `false`? The function does not handle this case and it triggered this line: https://github.com/svg/svgo/blob/77102ed0963a5f1597e8072536311b55465e1874/lib/svgo.js#L21

I encountered this issue on parcel 2.0.0-rc.0. I have SVG, but I don't have a corresponding SVG Optimizer config. I do not have .parcelrc overrides.

Reply:
I've opened https://github.com/svg/svgo/pull/1573. Sounds like you could be seeing what I've just found, which is actually https://github.com/svg/svgo/blob/721a1e591da131affece29362cf7e7089b474904/lib/svgo-node.js#L57-L59 (which doesn't have the null-ish check).